### PR TITLE
GUAC-1263: Add Italian on screen keyboard layout and missing translations

### PR DESCRIPTION
**Pull request by IT-Experte**

### CODE REVIEW

File: guacamole/src/main/webapp/layouts/it-it-qwerty.json [hunk 1]
```diff
@@ -0,0 +1,453 @@
...
+            "modifier" : "shift",
+            "keysym"   : 65506
+        }],
+        "LCtrl" : [{
+            "title"    : "Strg",
+            "modifier" : "control",
+            "keysym"   : 65507
+        }],
+        "RCtrl" : [{
+            "title"    : "Strg",
```
Comment (mike-jumper) [inline guacamole/src/main/webapp/layouts/it-it-qwerty.json]: Is this correct for Italian? I thought "Strg" was specific to German keyboards.

**Comment by IT-Experte** [comment]
Nop. It's not correct. Your are right. Sorry. I have checked it three times and youd found the mistake in one second. :-(

**Comment by mike-jumper** [comment]
No worries - that's why we have these reviews. Everyone benefits from extra eyes.

**Comment by IT-Experte** [comment]
I have fixed it now.

**Pull request merged**